pkg/apis/openstack/v1alpha1/types_infrastructure.go

```diff
@@ -43,6 +43,9 @@ type Networks struct {
 	Worker string `json:"worker"`
 	// Workers is a CIDRs of a worker subnet (private) to create (used for the VMs).
 	Workers string `json:"workers"`
```

Comment:
Hmm I guess in consequence of adding a choice (existing network or new network created by Gardener) we need to make this as well a `*string` and `+optional`.

I guess we need then also to validate that either `.workers` or `.name` is set. One of those is required.

Reply:
What we are specifying is the name of an existing Neutron network. The CIDR will then be used to create a new subnet in the network we previously specified so `Workers` is still a mandatory field IMHO.

Reply:
Ahh sorry. I mixed things. You are right. I thought `.workers` was the cidr for the network, but on OpenStack networks does not have an own cidr or at least we don't specify it.